grpc/proto/space.proto

```diff
@@ -294,6 +338,7 @@ message Bucket {
   int64 updatedAt = 5;
   repeated BucketMember members = 6;
   bool isPersonalBucket = 7;
+  bool backedUp = 8;
```

Comment:
Maybe `isBackupEnabled`? 🤔

Reply:
Fixed in [35ad3a0](https://github.com/FleekHQ/space-daemon/pull/109/commits/35ad3a05949cc481001b673672990921b33a4951).